README.md

```diff
@@ -16,6 +16,7 @@ There are three version sets of the connector available through Maven, a 2.4.x,
 | Spark 2.4.x compatible connnector | `com.microsoft.azure:spark-mssql-connector:1.0.2` | 2.11 |
 | Spark 3.0.x compatible connnector | `com.microsoft.azure:spark-mssql-connector_2.12:1.1.0` | 2.12 |
 | Spark 3.1.x compatible connnector | `com.microsoft.azure:spark-mssql-connector_2.12:1.2.0` | 2.12 |
+| Spark 3.3.x compatible connnector | `com.microsoft.azure:spark-mssql-connector_2.12:1.3.0` | 2.12 |
```

Comment:
will update to 2.13

Reply:
why did we change this. Scala version for Spark 3.3 is till 2.12